ext/standard/basic_functions.stub.php

```diff
@@ -1660,6 +1660,9 @@ function array_unshift(array &$array, mixed ...$values): int {}
 
 function array_splice(array &$array, int $offset, ?int $length = null, mixed $replacement = []): array {}
 
+/**
+ * @compile-time-eval
```

Comment:
This seems to be the cause of very strange test failure:
```
 ========DIFF========
--
       [1]=>
       int(4)
     }
013+ 
014+ Fatal error: Uncaught ArgumentCountError: var_dump() does not accept unknown named parameters in /home/runner/work/php-src/php-src/Zend/tests/named_params/internal.php:6
015+ Stack trace:
016+ #0 /home/runner/work/php-src/php-src/Zend/tests/named_params/internal.php(6): var_dump(Array, preserve_keys: true)
017+ #1 {main}
018+   thrown in /home/runner/work/php-src/php-src/Zend/tests/named_params/internal.php on line 6
013- array(1) {
014-   ["b"]=>
015-   int(1)
016- }
017- array(1) {
018-   ["b"]=>
019-   int(1)
020- }
021- string(6) "   foo"
022- int(3)
========DONE========
FAIL Named params on internal functions [Zend/tests/named_params/internal.phpt] 
```

it seems opcache/JIT has some problem with named param with internal/CTE functions, can someone please advise a fix?

Reply:
Anyone has an idea what is going on here?

Reply:
tracked in #10801